hyprpm/src/core/PluginManager.cpp

```diff
@@ -153,59 +157,59 @@ bool CPluginManager::addNewPluginRepo(const std::string& url, const std::string&
     m_szWorkingPluginDirectory = "/tmp/hyprpm/" + USERNAME;
 
     if (!createSafeDirectory(m_szWorkingPluginDirectory)) {
-        std::cerr << "\n" << Colors::RED << "✖" << Colors::RESET << " Could not prepare working dir for repo\n";
+        std::println(stderr, "\n{}", failureString("Could not prepare working dir for repo"));
         return false;
     }
 
-    progress.printMessageAbove(std::string{Colors::RESET} + " → Cloning " + url);
+    progress.printMessageAbove(infoString("Cloning ", url));
```

Comment:
ah dammmit, should I rebase everything again or you could just fix it in small commit?